scrapy/commands/fetch.py

```diff
@@ -30,15 +31,19 @@ def add_options(self, parser):
     def _print_headers(self, headers, prefix):
         for key, values in headers.items():
             for value in values:
-                print('%s %s: %s' % (prefix, key, value))
+                self._print_bytes(prefix + b' ' + key + b': ' + value)
 
     def _print_response(self, response, opts):
         if opts.headers:
-            self._print_headers(response.request.headers, '>')
+            self._print_headers(response.request.headers, b'>')
             print('>')
-            self._print_headers(response.headers, '<')
+            self._print_headers(response.headers, b'<')
         else:
-            print(response.body)
+            self._print_bytes(response.body)
+
+    def _print_bytes(self, bytes_):
+        bytes_writer = sys.stdout if six.PY2 else sys.stdout.buffer
+        bytes_writer.write(bytes_ + b'\n')
```

Comment:
When writing bytes to stdout (using ``sys.stdout.buffer.write``)
![2016-01-21 11 11 28](https://cloud.githubusercontent.com/assets/424613/12474901/c8378e22-c02f-11e5-8bf9-0958024737fb.png)